examples/er-rest-example/project-conf.h

```diff
@@ -60,6 +60,9 @@
 #undef NETSTACK_CONF_RDC
 #define NETSTACK_CONF_RDC              nullrdc_driver
 
+#undef RPL_CONF_MAX_DAG_PER_INSTANCE
+#define RPL_CONF_MAX_DAG_PER_INSTANCE     1
```

Comment:
This was added to reduce the code size so it could be compiled on Tmote Sky. Since I think the Travis test now compiles for WiSMote it might no longer be needed. I will remove it.